docs-chef-io/content/inspec/profiles/depends.md

```diff
@@ -360,3 +362,44 @@ generates an `inspec.lock` file.
 
 If you add or update dependencies in `inspec.yml`, dependencies may be re-vendored
 and the lockfile updated with `inspec vendor --overwrite`
+
+## Gem based Resource Packs dependencies
+
+Any profile dependent on gem-based resource packs can specify them using the `gem` setting in the `inspec.yml` metadata file.
+
+InSpec handles the installation and download of the specified resource pack gems when running the profile.
```

Comment:
Replace it with: InSpec handles installing and downloading the specified resource pack gems when running the profile.

Reply:
Better concerning?
( I would still suggest to change it to the once requested)